package.json

```diff
@@ -27,6 +27,7 @@
   "homepage": "https://github.com/juju/js-libjuju#readme",
   "devDependencies": {
     "btoa": "^1.2.1",
+    "documentation": "^8.1.2",
```

Comment:
Is this being used somewhere? Maybe I missed it...

Reply:
Yes: https://github.com/frankban/js-libjuju/blob/docs/generator/_render.py#L161